.github/actions/Ubuntu18.04-build/debian/control

```diff
@@ -21,15 +21,15 @@ Build-Depends: autotools-dev, ghostscript, gsfonts, libjbig-dev:amd64,
 Standards-Version: 4.4.1
 Homepage: https://imagemagick.org
 
-Package: imagemagick
+Package: imagemagick-alfresco
 Architecture: amd64
 Depends: ${misc:Depends}, ${shlibs:Depends}
 Description: ImageMagick 7 for Alfresco Transformation service
  This is the vanilla ImageMagick 7 software packaged for automated
  Alfresco deployments. All credit goes to ImageMagick Studio LLC, their
  contributors and community.
```

Comment:
if we are not changing PATH to make bin available system-wide then it could make sense to not use `Provides: imagemagick`. Anyway I am not sure if it make sense to install the standard imagemagick alongside with our custom package and this could be an effective way to prevent that.